### DISCUSSION TITLE
Java gRPC client ACL not working

### DISCUSSION THREAD

**Discussion by guyu0000** [apache/rocketmq#8317] [General]
version:
  server: 5.2.0
  rocketmq-client-java: 5.0.7 (grpc)
  rocketmq-client: 5.2.0 (Remoting)

When ACL is enabled on the broker, it works on the dashboard and also for Remoting protocol clients. However, it does not work for gRPC clients, which can still send and consume messages. Is this a bug or feature?

**Comment by dingshuangxi888** [comment]
Can you show some logs in gRPC clients.

**Comment by dingshuangxi888** [comment]
The gRPC client can only connect to the rocketmq-proxy server, please check that you did not connnect to the broker.

**Reply by guyu0000** [reply]
Confirmed that the connection is to the proxy server. As you mentioned, the gRPC client can only connect to the proxy server. Do I need to configure anything for ACL on the proxy server?
